FlowCryptUI/Cell Nodes/InboxCellNode.swift

```diff
@@ -137,52 +177,62 @@ public final class InboxCellNode: CellNode {
     }
 
     override public func layoutSpecThatFits(_: ASSizeRange) -> ASLayoutSpec {
-        let emailElement: ASLayoutElement = {
-            guard let countNode else { return emailNode }
-            emailNode.style.flexShrink = 1.0
-            let spec = ASStackLayoutSpec.horizontal()
-            spec.children = [emailNode, countNode]
-            spec.spacing = 5
-            return spec
-        }()
+        let emailElement = ASStackLayoutSpec.horizontal()
+        emailElement.spacing = 5
+        emailElement.children = countNode == nil ? [emailNode] : [emailNode, countNode!]
 
+        // Create Name-Location Stack
         let nameLocationStack = ASStackLayoutSpec.vertical()
-        nameLocationStack.spacing = 4
+        nameLocationStack.spacing = Constants.topBottomVerticalSpacing
         nameLocationStack.style.flexShrink = 1.0
         nameLocationStack.style.flexGrow = 1.0
-        separatorNode.style.flexGrow = 1.0
+
+        let messageBadgeStack = ASStackLayoutSpec.horizontal()
+        messageBadgeStack.children = [messageNode, badgeNode]
+        messageBadgeStack.alignItems = .center
+        messageBadgeStack.spacing = 6
+
+        // Configure message and badge
+        nameLocationStack.children = input.badgeText == nil
+            ? [emailElement, messageNode]
+            : [emailElement, messageBadgeStack]
+
+        // Setup separator
         separatorNode.style.preferredSize.height = 0.5
 
-        if input.badgeText != nil {
-            messageNode.style.flexShrink = 1.0
-            let messageStack = ASStackLayoutSpec.horizontal()
-            messageStack.style.flexShrink = 1.0
-            messageStack.alignItems = .center
-            messageStack.spacing = 6
-            messageStack.children = [messageNode, badgeNode]
-            nameLocationStack.children = [emailElement, messageStack]
-        } else {
-            nameLocationStack.children = [emailElement, messageNode]
-        }
+        let iconsSpec = ASStackLayoutSpec.horizontal()
+        iconsSpec.spacing = 3
+        iconsSpec.children = [
+            input.hasAttachment ? attachmentIcon : nil,
+            input.hasPublicKey ? publicKeyIcon : nil,
+            input.isEncrypted ? encryptedIcon : nil
```

Comment:
<img width="84" alt="Screenshot 2023-09-18 at 12 48 18" src="https://github.com/FlowCrypt/flowcrypt-ios/assets/1062093/4a2bca3d-9802-42eb-a2b6-d20eb25c531d">

Looks like there are too many icons when thread contains attachments and public key.

Let's show only `public key` icon when thread contains only public key, and only `attachment` icon when thread contains attachments and public keys

Reply:
Well, what if email contains public key attachment and other attachment files?

Reply:
We need show both icons (public key icon and attachment icon) for only this case?

Reply:
No, let's show only `attachment` icon in this case. So `public key` icon will be shown when thread contains only public key attachment

Reply:
OK